### ISSUE TITLE
Add term enlarged distal tibial epiphysis

### ISSUE THREAD

**Issue by lisannesw** [obophenotype/human-phenotype-ontology#10379]
**Preferred term label:**
Enlarged distal tibial epiphysis

**Synonyms**
Large distal tibial epiphysis

**Definition (free text, please give PubMed ID)**
Enlarged distal tibial epiphysis
PMID: 12624140

**Parent term (use hpo.jax.org/app)**
Abnormality of the distal tibial epiphysis HP:0010592

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**
ACROCAPITOFEMORAL DYSPLASIA; ACFD, OMIM607778

**Your nano-attribution (ORCID)**
Andreas Zankl: 0000-0001-8612-1062, @azankl
Lisanne Wit: 0009-0007-1020-5412

**Comment by pnrobinson** [comment]
@azankl

In this article it is the distal femoral but not tibial that are enlarged?
https://pubmed.ncbi.nlm.nih.gov/12624140/

**Comment by pnrobinson** [comment]
@lisannesw -- is this tibial or femoral? In any case I added the term, please reopen if needed

Adding new term: 
- ID: HP:6000886
- label: Enlarged distal tibial epiphysis
- parents: [Abnormality of the distal tibial epiphysis (HP:0010592)](https://hpo.jax.org/app/browse/term/HP:0010592)
- pmid: PMID:12624140
- def: Size (volume) of the distal epiphysis of the tiba above the upper limit of normal.
- comment: 
- orcid: orcid:0000-0002-0736-9199
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/10379
- synonyms: Large distal tibial epiphysis(NONE)

**Comment by azankl** [comment]
@pnrobinson sorry, did not reply quick enough to your question. The distal femoral epiphyses is certainly large, but in the text, they also say the distal tibial epiphysis is large. The image is blurry, so cannot really tell if the distal tibial epiphysis is enlarged. Since its mentioned in the text, OK to add it.
